_articles/upgrade-pop.md

```diff
@@ -21,53 +22,60 @@ section: pop-ubuntu
 
 ---
 
-Pop!_OS 18.10 was released in October, 2018. This article explains how to upgrade from Pop!_OS 18.04 to Pop!_OS 18.10.
+### Upgrading to Pop!\_OS 19.10
 
-Pop!_OS 19.04 was released in April, 2019. This article explains how to upgrade from Pop!_OS 18.10 to Pop!_OS 19.04.
+> NOTE: Users running Pop!_OS 18.04 LTS may also upgrade to Pop!_OS 19.10 (if they wish)
 
-### Create Installation USB
 
-First, we recommend having Pop!_OS installation media handy before starting any upgrades in case you need to reinstall the operating system. Instructions on creating a USB with Pop!_OS are located in our [live disk](/articles/live-disk/) article.
+First, make sure you have applied all updates to your system. You can do this through the Pop!\_Shop, or through Terminal.
```

Comment:
Please also have the notes about how to get here without a notification

Reply:
Added steps to find the Upgrade button without the notification.